### PR TITLE
azure scripts

### PR DESCRIPTION
**Pull request by himatejvaltix**

### CODE REVIEW

File: discovery.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    tmp_sub_list=($sub_list)
+    echo "Setting the subscription to ${tmp_sub_list[$sub_selection]} / ${id_list[$sub_selection]}"
+    # az account set --subscription ${id_list[$sub_selection]} --only-show-errors
+    account_info=$(az account show -s ${id_list[$sub_selection]})
+    sub_name=$(echo $account_info | jq -r .name)
+    sub_id=$(echo $account_info | jq -r .id)
+    unset IFS
+fi
+
+EVENT_SUB_NAME=vtxcontroller-discovery
```
Comment (himatejvaltix) [inline discovery.sh]: TODO: fix this

File: discovery.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+echo "Location       : ${location}"
+echo "Webhook Endpoint : ${webhook_endpoint}"
+echo "Subscription ID: ${sub_id}"
+
+read -p "Continue creating? [y/n] " -n 1
+echo ""
+if [[ "$REPLY" != "y" ]]; then
+    exit 1
+fi
+
```
Comment (himatejvaltix) [inline discovery.sh:92]: TODO: create resource group ValtixRG

File: discovery.sh [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+echo "create storage account $STORAGE_ACCT_NAME"
+az storage account create --name $STORAGE_ACCT_NAME --resource-group $resourcegroup --location $location
```
Comment (himatejvaltix) [inline discovery.sh]: TODO: if storage account name already exists throw error

Comment (himatejvaltix) [reply discovery.sh]: TODO: print storage acccount id

File: azure-setup.sh [hunk 4]
```diff
@@ -1,22 +1,27 @@
 #! /bin/bash
```
Comment (himatejvaltix) [inline azure-setup.sh:1]: TODO: fix file permissions

**Comment by michaelvaltix** [comment]
Please add "Microsoft.Network/networkWatchers/*", in the permission section.

**Review by michaelvaltix — APPROVED** [review]

**Pull request merged**